tethys_apps/decorators.py

```diff
@@ -102,7 +102,8 @@ def my_controller(request):
     """  # noqa: E501
 
     use_or = kwargs.pop('use_or', False)
-    message = kwargs.pop('message', "We're sorry, but you are not allowed to perform this operation.")
+    message = kwargs.get('message', "We're sorry, but you are not allowed to perform this operation.")
+    production_message = kwargs.get('message', "We're sorry, but the operation you requested cannot be found.")
```

Comment:
I think just having one message is sufficient. I don't think to distinguish between dev and production mode.

Reply:
I have made the suggested change.